src/rules/npmNamingRule.ts

```diff
@@ -221,18 +201,82 @@ function walk(ctx: Lint.WalkContext<Options>): void {
                         break;
                 }
             }
-
         } catch (e) {
             // We're ignoring exceptions.
         }
     }
     // Don't recur, we're done.
 }
 
+const enabledSuggestions: ExportErrorKind[] = [
+    ErrorKind.JsPropertyNotInDts,
+    ErrorKind.JsSignatureNotInDts,
+];
+
+function toOptionsWithSuggestions(options: CriticOptions): CriticOptions {
```

Comment:
this is almost a superset of toCriticOptions (I'm not sure if `{ ...options }` copies more properties than `{ mode: options.mode }`. Are both really needed?

Reply:
They are meant to do different things. This function just modifies the dts-critic options so that we get the kinds of errors we want to use as suggestions.
`toCriticOptions` just transforms the options format as written in the tslint config to a options format that dts-critic understands/is more convenient (by converting from an array to a map).